### PR TITLE
[MODULAR]Gives new Lungs as Selectable on the augmentation Menu

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request:

This PR gives four newly selectable lungs to choose from.
Heat Adapted Lungs
Toxin Adapted Lungs
Low-Pressure Adapted Lungs
Cold Adapted Lungs

image

## Why It's Good For The Game
Mostly news ways to play and making your species be more believable!
Its don't make sense polar creatures are not adapted to cold temperatures same as hotter ones!

## Changelog
:cl:
add: Adds 4 new selectable lungs on the augmentation loadout menu.

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
image
I also organized the organs on the DMI!

**Comment by Guidesu** [comment]
i tested everything its works.
The sprites and kinds of stuff are all goodie!

**Comment by Guidesu** [comment]
Still needs reviewing for names of organs and such

**Comment by YakumoChen** [comment]
Are these going to be blacklisted for species with different atmos requirements? Being able to be plasman with regular lungs at roundstart seems kind of bad.

**Comment by Guidesu** [comment]
i forgot about this

**Comment by Guidesu** [comment]
oh well.
Plasma mans cant really pick anything
image

**Comment by Guidesu** [comment]
image
And its seems vox can pick normal lungs on normal server

**Comment by Vizoplez** [comment]
Codes good, thumbsup from an intern dev; I'd say these are also rather balanced since even basic cybernetic lungs are pretty decent.

Edit; Though, I recommend you modularise this.

**Comment by Guidesu** [comment]
i made modular me thinks

File: modular_skyrat/organs/lungs.dm [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	heat_damage_type = BURN
+
+/obj/item/organ/lungs/toxin
+	name = "Toxic Adapted Lungs"
+	desc = "These lungs seem to be adapted to Toxic environments. These lungs seem fairly less resistant to low temperatures and high ones."
+	icon_state = "lungs_toxin"
+	safe_toxins_max = 27
+	safe_co2_max = 27
+
+	cold_message = "Your face is melting due to the freezing temperature!"
```
Comment (tf-4) [inline modular_skyrat/organs/lungs.dm]: make it clearer that it's the lungs

```suggestion
	cold_message = "You can't stand the freezing cold with every breath you take!"
```

File: modular_skyrat/organs/lungs.dm [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	cold_level_1_threshold = 248
+	cold_level_2_threshold = 220
+	cold_level_3_threshold = 170
+	cold_level_1_damage = COLD_GAS_DAMAGE_LEVEL_2 //Keep in mind with gas damage levels, you can set these to be negative, if you want someone to heal, instead.
+	cold_level_2_damage = COLD_GAS_DAMAGE_LEVEL_2
+	cold_level_3_damage = COLD_GAS_DAMAGE_LEVEL_3
+	cold_damage_type = BURN
+
+
+	hot_message = "You cant stand the searing heat on your face!"
```
Comment (tf-4) [inline modular_skyrat/organs/lungs.dm]: make it clearer that it's the lungs

```suggestion
	hot_message = "You can't stand the searing heat with every breath you take!"
```

File: modular_skyrat/organs/lungs.dm [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	heat_level_2_threshold = 348
+	heat_level_3_threshold = 1000
+	heat_level_1_damage = HEAT_GAS_DAMAGE_LEVEL_2
+	heat_level_2_damage = HEAT_GAS_DAMAGE_LEVEL_2
+	heat_level_3_damage = HEAT_GAS_DAMAGE_LEVEL_3
+	heat_damage_type = BURN
+
+/obj/item/organ/lungs/oxy
+	name = "Low-Oxygen Adapted Lungs"
+	desc = "These lungs seem to be adapted to Low Oxygen environments. These lungs seem fairly less resistant to low temperatures and high ones."
```
Comment (tf-4) [inline modular_skyrat/organs/lungs.dm]:
```suggestion
	desc = "A set of lungs adapted to lower-pressure environments, though more susceptible to extreme temperatures."
```

File: modular_skyrat/organs/lungs.dm [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+	icon_state = "lungs_toxin"
+	safe_oxygen_min = 5
+
+	hot_message = "You cant stand the searing heat on your face!"
```
Comment (tf-4) [inline modular_skyrat/organs/lungs.dm]: make it clearer that it's the lungs

```suggestion
	hot_message = "You can't stand the searing heat with every breath you take!"
```

File: modular_skyrat/organs/lungs.dm [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+	heat_level_1_threshold = 318
+	heat_level_2_threshold = 348
+	heat_level_3_threshold = 1000
+	heat_level_1_damage = HEAT_GAS_DAMAGE_LEVEL_2
+	heat_level_2_damage = HEAT_GAS_DAMAGE_LEVEL_2
+	heat_level_3_damage = HEAT_GAS_DAMAGE_LEVEL_3
+	heat_damage_type = BURN
+
+	cold_message = "Your face is melting due to the freezing temperature!"
```
Comment (tf-4) [inline modular_skyrat/organs/lungs.dm]: make it clearer that it's the lungs

```suggestion
	cold_message = "You can't stand the freezing cold with every breath you take!"
```

**Review by tf-4 — COMMENTED** [review]
general grammar review. Code seems fine.

File: tgstation.dme [hunk 6]
```diff
@@ -3908,4 +3908,5 @@
 #include "modular_skyrat\modules\verbs\code\modules\mob\say.dm"
 #include "modular_skyrat\modules\verbs\code\modules\mob\subtle.dm"
 #include "modular_skyrat\modules\xenomorph\code\xenomorph_balance.dm"
+#include "modular_skyrat\organs\lungs.dm"
```
Comment (Gandalf2k15) [inline tgstation.dme]: You're gonna need to put the files in the modules folder, see above

Comment (Guidesu) [reply tgstation.dme]: doing it

Comment (Guidesu) [reply tgstation.dme]: done

**Comment by Guidesu** [comment]
pain i will need to check these errors

**Comment by Guidesu** [comment]
image
It WAS JUST NEEDED TO ENABLE THE DAMN FILE!
REEEEE

**Comment by NullDagaf** [comment]
Time to watch it get powergamed to shit.
Will be interesting at least.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**